src/service/runnerService.ts

```diff
@@ -62,11 +62,12 @@ export class RunnerService {
         const pos = currentPath.lastIndexOf('.');
         const compilePath =
             currentPath.substr(0, pos < 0 ? currentPath.length : pos) + '.exe';
+        const guiKey = showGui ? '/gui' : '';
         if (
             await Process.exec(
                 `"${Global.getConfig(
```

Comment:
Let's see if we can create a pure helper function that generates this string that's passed to `Process.exec`. We can use the internal-external pattern from the formatter. Then we can just test the pure helper function and call it tested :)

Reply:
Done